migration-roadmap/badge.md

```diff
@@ -0,0 +1,337 @@
+
+| CSS selector                   | Attribute or slot       | Status          |
+| ------------------------------ | ----------------------- | --------------- |
+| `.spectrum-Badge--accent`      | `variant="accent"`      | Implemented     |
+| `.spectrum-Badge--blue`        | `variant="blue"`        | Implemented     |
+| `.spectrum-Badge--celery`      | `variant="celery"`      | Implemented     |
+| `.spectrum-Badge--chartreuse`  | `variant="chartreuse"`  | Implemented     |
+| `.spectrum-Badge--cyan`        | `variant="cyan"`        | Implemented     |
+| `.spectrum-Badge--fuchsia`     | `variant="fuchsia"`     | Implemented     |
+| `.spectrum-Badge--gray`        | `variant="gray"`        | Implemented     |
+| `.spectrum-Badge--green`       | `variant="green"`       | Implemented     |
+| `.spectrum-Badge--indigo`      | `variant="indigo"`      | Implemented     |
+| `.spectrum-Badge--informative` | `variant="informative"` | Implemented     |
+| `.spectrum-Badge--magenta`     | `variant="magenta"`     | Implemented     |
+| `.spectrum-Badge--negative`    | `variant="negative"`    | Implemented     |
+| `.spectrum-Badge--neutral`     | `variant="neutral"`     | Implemented     |
+| `.spectrum-Badge--notice`      | `variant="notice"`      | Implemented     |
+| `.spectrum-Badge--orange`      | `variant="orange"`      | Implemented     |
+| `.spectrum-Badge--positive`    | `variant="positive"`    | Implemented     |
+| `.spectrum-Badge--purple`      | `variant="purple"`      | Implemented     |
+| `.spectrum-Badge--red`         | `variant="red"`         | Implemented     |
+| `.spectrum-Badge--seafoam`     | `variant="seafoam"`     | Implemented     |
+| `.spectrum-Badge--yellow`      | `variant="yellow"`      | Implemented     |
+| `.spectrum-Badge--brown`       | `variant="brown"`       | Missing from WC |
+| `.spectrum-Badge--cinnamon`    | `variant="cinnamon"`    | Missing from WC |
+| `.spectrum-Badge--pink`        | `variant="pink"`        | Missing from WC |
+| `.spectrum-Badge--silver`      | `variant="silver"`      | Missing from WC |
+| `.spectrum-Badge--turquoise`   | `variant="turquoise"`   | Missing from WC |
+
+## Summary of changes
+
+### CSS => SWC changes
+
+**Missing from WC:**
+
+- **New color variants**: `brown`, `cinnamon`, `pink`, `silver`, `turquoise`
+- **Style variants**: `outline` and `subtle` styles
+
+**Note**: Fixed positioning exists in both SWC and CSS but is not in the design spec. Consider whether to keep this for 2nd gen.
+
+### CSS DOM structure changes
+
+**No significant structural changes.** Only difference is the addition of `spectrum-Badge--style-default` class in Spectrum 2, which explicitly declares the default style for consistency with new outline/subtle variants.
+
+## Resources
+
+- [CSS migration](https://github.com/adobe/spectrum-css/pull/3740)
+- [Spectrum 2 preview](https://spectrumcss.z13.web.core.windows.net/pr-2352/index.html?path=/story/components-badge--default)
```

Comment:
Looks like this is the only one going to a default story page, as opposed to the CSS docs page.